pkg/cmd/cli/describe/projectstatus_test.go

```diff
@@ -137,6 +137,8 @@ func TestProjectStatus(t *testing.T) {
 				"database deploys",
 				"frontend deploys",
 				"with docker.io/openshift/ruby-20-centos7:latest",
+				"#2 deployment failed less than a second ago: unable to contact server - 0/1 pods",
+				"#2 deployment running for 7 seconds - 2/1 pods",
 				"#1 deployed 8 seconds ago",
```

Comment:
I'm mostly concerned about scanability.  The goal is to find the # as fast as possible visually.  For right now, that's best done on the left margin of the text.